content/collections/data/en/chrome-extension-debug.md

```diff
@@ -75,7 +75,7 @@ If all checks pass but guides or surveys still don't work, verify these settings
 
 ### Troubleshooting Guides & Surveys
 
-The troubleshooting section shows all published guides and surveys that appear on the current page. Each guide or survey displays its trigger conditions and whether those conditions pass.
+The troubleshooting section shows all published guides and surveys for the associated API key (project). Each guide or survey displays its trigger conditions and whether those conditions pass.
```

Comment:
```suggestion
The troubleshooting section shows all published guides and surveys for the associated project's API key. When you use Preview, all draft guides and surveys are visible. Each guide or survey displays its trigger conditions and whether those conditions pass.
```

Reply:
implemented your suggested change for the first sentence though!